src/pybind/mgr/cephadm/module.py

```diff
@@ -249,6 +255,12 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             'default': None,
             'desc': 'Custom repository password'
         },
+        {
+            'name': 'use_repo_digest',
+            'type': 'bool',
+            'default': False,
+            'desc': 'Automatically convert image tags to image digest. Make sure all daemons use the same image',
```

Comment:
The description sounds like the behavior users would expect to be active by default, so I'm confused why the default is set to "False".

Reply:
Eh, the description describes the flag itself. Independent of the value.

Reply:
+1 to use true as default value. Is the safe option!